### ISSUE TITLE
plugin not working

### ISSUE THREAD

**Issue by marvinNL** [snowgears/shopbugs#7]
here is the video and video proof
https://www.youtube.com/watch?v=G-8IVw0ydD4&feature=youtu.be

**Comment by snowgears** [comment]
It appears you are using the plugin on PaperSpigot, which will not work for custom registration of commands (which this plugin uses to register the 'shop' command)

**Comment by marvinNL** [comment]
"It appears you are using the plugin on PaperSpigot, which will not work for custom registration of commands (which this plugin uses to register the 'shop' command)"
if that's true then please refund me if I'm unable to use the plug-in then there is no point in continuing discussing about it I would appreciate it if you can refund me since I'm unable to use the plug-in

**Comment by BlisteringSyko** [comment]
I'm gonna see if i can maybe help you with this. Im running Paper spigot, version 486 (1.13.2), This plugin loads and works fine, including the /shop commands.
I'm trying to go back through the shop thread on spigot but i might miss one of your posts, in case i'm asking you to do something that you have already tried.

As far as i know latest version wont work under 1.12.2, You need either a a 1.13 build of Spigot or a 1.13 build of one of its forks, such as Paper.

This will not work under forge, its a different framework than bukkit/spigot.

I suggest going to this link and downloading the latest paper build, stick in an a new empty folder and run it, let it generate a fresh set of the server files, stick the plugin in the plugins folder and restart the server. I just want to know if the plugin works for you on in a clean install.
[https://papermc.io/ci/job/Paper-1.13/](url) 

If it doesn't work, please post the latest.log that contains any exceptions.

**Comment by marvinNL** [comment]
> I watched the video, you're using 1.8.0.1 on a 1.12.2 server, which is not compatible. See #11.

then you did not clearly see the video I recommend watching it again and look closely

**Comment by BlisteringSyko** [comment]
Please Start the server you used in the video and post the latest.log in the /log folder. Its too hard to see whats being output in the cmd window.   I cant see what version of the plugin you are using in that video, all i can tell is that you are running 1.12.2 and that something caused 2 errors to be sent to the console, the only word i can make out in one of the errors is null. 

I need to see the errors that come up on startup, to see if the plugin is loading properly.

We have no clue what version of the plugin you downloaded. If you just clicked the download button on the main page and not one of the files under "Version History" then you downloaded the version for 1.13 which is not compatable with any version under, ie: 1.12.2 or as you mentioned you tried a 1.8 server.

If you downloaded the version for 1.13 you must be running Spigot 1.13 or a 1.13 fork of spigot such as paper, not bukkit, not craftbukkit, not forge, not vanilla.


The Api in 1.13 is different from the previous versions, a plugin made for 1.13 will not work in 1.12.

Please post the latest.log here in text.

**Comment by marvinNL** [comment]
I am unable to do that 
I have deleted everything 
and since I am no longer able or allowed to re-download there's no chance for me to do it
when I was able to download I made sure to run the proper version I am not that f****** retarded
I also tried multiple versions as I also explained in the video 
I am willing to give it another try however you would have to give me access to be able to re-download
or better yet build a entire server for me put it in a zip or WinRAR and send it to me so I only have to press the start button to start server make sure the server has at least 8  GB to work with preferred server version 1.12.2

**Comment by untuned** [comment]
![capture2](https://user-images.githubusercontent.com/7635652/50836306-eac56200-1326-11e9-984a-27e89d416cfe.PNG)
**1.8.0.1 IS NOT COMPATIBLE WITH 1.12.2.**

**Comment by BlisteringSyko** [comment]
I tried to help, but if you deleted everything then there's nothing any of us can do at this point. I'm not going to share a copy of the plugin without @snowgears permission.

Perhaps if you talk to spigot support they may be able to help you with the entire payment thing since you say you canceled the refund.

Also, if your going to be hosting your own servers, you really need to understand at least the basics, and differences between versions a lot changed in 1.13 from 1.12, as well as the difference between spigot and forge. 

Trying to run a spigot plugin on a forge server or a vanilla server from mojang is like trying to run diesel in a gas engine. Then complaining to the dealership that they sold you a messed up car. And then refusing to understand whats actually wrong.

I dont mind helping, but im not going to spoon feed some one who cant be rational when people are trying to help.

**Comment by marvinNL** [comment]
( commented 22 days ago ) do you really think I'm going to wait this long with crap that doesn't work 

the particular version may not work on that particular  version however  as I mentioned in the video I tried multiple different servers and different spigot servers if you download 3 different versions of the mod then I wouldn't be surprised if I mix one up here and there 

5 different versions of Minecraft ( 1.7 / 1.8 / 1.11 / 1.12 / 1.13 )
+
3 different versions of the mod ( shop 1.12 / shop 1.8 / shop 1.11 )
+
4 different instances of the server ( spigot 5x /  forge server 5 x / vanilla server 5x / craftbukkit 5x )

that is roughly 60 servers and all of them did not work
if I include the video that would be 61 servers 

do you really think that after all of that I will keep the file ? 
on top of that I've waited a very long time on top of that
it is bad enough I have trouble reading and writing due to my dyslexia

when I asked for the creators help I got no reply for a very long time 
when he finally replied to me he said that I had to cancel my refund in order to get help so I did
but now I haven't heard from him since then which put me in a very very angry angry mood
since I cancelled the refund I am no longer able to do anything and I still have not heard from the Creator which pisses me off even more it has been 22 days now and it's still not fixed I've just given up on the hole situation if I have to wait this long in order to get help then I'm sorry but then I hope you can understand my frustration I've been trying to be polite as possible after spending 2 weeks trying to get this damn thing to work it is becoming more and more difficult to maintain my patience long story short I paid for something and it's not working and I'm not getting any help unless you wait about a full month
I purchased this on
10 December 2018
 I have been waiting long enough and there's nothing I can do about it because certain people just don't give a ( %^$#* )
how do you expect me to feel that I would be happy about it ? or just keep the files and wait for a miracle ?
how do you expect me to react to that no normal human will stay calm unless they think drugs do you want me to take drugs ? 
I mean come on 
so I hope you can understand what I'm going through 
and specially when people are trying to breathe down my neck
make it simple if I can't host the server I won't make any money
to buy food to survive 
I am a personal host / I am not a professional / I am not a company / I do what I have to do to survive 
for example hosting service for people in order to get some money

I am a handicapped person living on benefits from the government
I have lost my job and I'm unable to work due to my physical body condition
I've been sitting at home since 2004 with no job to speak off
due to my disability / lower back injuries 
I am no longer able to work however the income that are received from the government is not enough
to pay the bills and my medication therefore I am hosting service in order to get a little bit of extra cash
the group that I belong to ( gamernology ) is a startup group and we do not have that much influence
and to make matters worse due to my physical brain damage I am unable to function properly which is another factor why I can't work anymore
currently I'm trying to negotiate with a game developer company 
which is going pretty well this will not be a paid job however we will receive reputation for it 
and my name will be in the credits once the game gets released which hopefully be at the end of this year

current status is pre alpha to Alpha
as a pre alpha tester I have a lot of stuff to do 
so in other words if I cannot receive any help then I will not be bothered to promote this plug-in

and especially with all the crap that has been happening so again I don't care who fixes it as long as it is getting fixed
